config/samples/multiple-auth/httpbin_basic_authentication_and_mTLS.yaml

```
@@ -0,0 +1,24 @@
+# Here we are creating an API definition with basic authentication and mTLS with basic authentication as base identy for httpbin 
+
+apiVersion: tyk.tyk.io/v1alpha1
+kind: ApiDefinition
+metadata:
+  name: httpbin-multiple-authentications
+spec:
+  name: Httpbin Multiple Authentications
+  protocol: http
+  active: true
+  base_identity_provided_by: BasicAuthUser
```

```suggestion
  base_identity_provided_by: basic_auth_user
```
While trying out this sample file I realized `BasicAuthUser` is not supported value.
Supported values are documented [here](https://tyk.io/docs/tyk-apis/tyk-gateway-api/oas/x-tyk-oas-doc/#authentication)